src/main/java/no/nav/familie/integrasjoner/kodeverk/KodeverkInitializer.kt

```diff
@@ -3,10 +3,12 @@ package no.nav.familie.integrasjoner.kodeverk
 import org.slf4j.LoggerFactory
 import org.springframework.boot.context.event.ApplicationReadyEvent
 import org.springframework.context.ApplicationListener
+import org.springframework.context.annotation.Profile
 import org.springframework.scheduling.annotation.Scheduled
 import org.springframework.stereotype.Component
 
 @Component
+@Profile("!dev & !integrasjonstest")
```

Comment:
Kjenner ikkje til korleis vi bruker kodeverk, kva er grunnen til at vi ikkje treng denne i dev?

Reply:
Den bare populererer en cache og går mot en url man ikke uansett når. Skaper bare støy

Reply:
Og det gjør den også for alle testene som starter opp disse 2 profilene